tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -325,11 +327,17 @@ def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_ex
                 if testing_mode == DUAL_TOR_MODE:
                     standby_duthost = rand_unselected_dut
                     start_dhcp_monitor_debug_counter(standby_duthost)
+                    client_iface = str(dhcp_relay['client_iface']['name'])
+                    # In case of active-standby ports northbound  trafic is duplicated
+                    # to both ToRs by the y-cable whereas is case of active-active
+                    # ports the traffic is ECMPd, therefore only one ToR receives
+                    # northbound packets
+                    discReqCount = 0 if client_iface in active_active_ports else 1
```

Comment:
client_iface is vlan member, I am not sure whether there is a scenario that the interface is not active_active_ports?

Reply:
The fixture active_active_ports: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/dualtor/dual_tor_common.py#L71-L81, returns a collection of ports with active-active cable type or more simply put as the ports connected to smart NIC through DAC. ( all ports in dualtor-aa and some ports in dualtor-mixed )

active_standby_ports: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/dualtor/dual_tor_common.py#L84-L94, returns a collection of ports with active-standby cable type of ports connected to the servers through y-cable. ( all ports in dualtor and some ports in dualtor-mixed )